Makefile.win

```diff
@@ -199,6 +208,49 @@ clean_crystal: ## Clean up crystal built files
 clean_cache: ## Clean up CRYSTAL_CACHE_DIR files
 	$(call RMDIR,"$(shell .\bin\crystal env CRYSTAL_CACHE_DIR)")
 
+.PHONY: rc
+rc: ## Write compiler resource script to standard output
+	$(eval comma := ,)
+	$(eval ver_comma := $(subst .,$(comma),$(subst -dev,,$(CRYSTAL_VERSION))))
+	$(eval build_date := $(shell git show -s --format=%cs HEAD))
+	$(eval build_year := $(firstword $(subst -, ,$(build_date))))
+	$(eval ver_full := $(CRYSTAL_VERSION)$(if $(CRYSTAL_CONFIG_BUILD_COMMIT), [$(CRYSTAL_CONFIG_BUILD_COMMIT)])$(if $(build_date), ($(build_date))))
+	@setlocal EnableDelayedExpansion &\
+	echo #include ^<windows.h^>&\
+	echo.&\
+	set "_dir=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))" &\
+	echo 1 ICON ^"!_dir:/=\\!src\\compiler\\crystal\\tools\\playground\\public\\favicon.ico^"&\
+	echo.&\
+	echo VS_VERSION_INFO VERSIONINFO&\
+	echo     PRODUCTVERSION $(ver_comma),0&\
+	echo     FILEVERSION    $(ver_comma),1&\
+	echo     FILEFLAGSMASK  VS_FFI_FILEFLAGSMASK&\
+	echo     FILEFLAGS      $(if $(findstring -dev,$(CRYSTAL_VERSION)),VS_FF_PRERELEASE,0)&\
+	echo     FILEOS         VOS_NT_WINDOWS32&\
+	echo     FILETYPE       VFT_APP&\
+	echo     FILESUBTYPE    VFT2_UNKNOWN&\
+	echo BEGIN&\
+	echo     BLOCK "StringFileInfo"&\
+	echo     BEGIN&\
+	echo         BLOCK "040904B0"&\
+	echo         BEGIN&\
+	echo             VALUE "ProductName",      "Crystal"&\
+	echo             VALUE "ProductVersion",   "$(ver_full)"&\
+	echo             VALUE "FileVersion",      "$(ver_full)"&\

```

Comment:
Is this correct? File version in the screenshot shows `1.6.0.1`, not the value of `ver_full`.

Reply:
Yeah this is correct, the dialog just displays the `FILEVERSION` field above but it looks like this field here is also required.